charts/rstudio-pm/templates/deployment.yaml

```diff
@@ -124,6 +124,9 @@ spec:
             {{- toYaml . | nindent 10 }}
           {{- end }}
         {{- end }}
+      {{- if .Values.extraContainers }}
+      {{- toYaml .Values.extraContainers | nindent 6 }}
```

Comment:
A bit of a nit, but indenting this one smidge further can make it easier to read the control flow

@atheriel do you have thoughts on whether to treat this like a template or not?

Reply:
Convention seems to be not to, I agree. Is it problematic to add later if requested?

Reply:
Hey guys. Do I need to fix anything here?

Reply:
A bit of a nit, but if you could indent this line two more characters, that would be appreciated! Other than that, LGTM!

Reply:
Please check if the latest push is like you requested.